site/site.go

```diff
@@ -17,6 +19,15 @@ import (
 	"golang.org/x/xerrors"
 )
 
+type apiResponseContextKey struct{}
+
+// WithAPIResponse returns a context with the APIResponse value attached.
+// This is used to inject API response data to the index.html for additional
+// metadata in error pages.
+func WithAPIResponse(ctx context.Context, apiResponse APIResponse) context.Context {
+	return context.WithValue(ctx, apiResponseContextKey{}, apiResponse)
+}
```

Comment:
Is this dev url specific or is it related to the error message discussion from yesterday?